eftest/src/eftest/runner.clj

```diff
@@ -75,20 +75,34 @@
         each-fixtures (-> ns meta ::test/each-fixtures test/join-fixtures)
         test-var      (-> (fn [v]
                             (when-not (and fail-fast? (failed-test?))
-                              (each-fixtures
-                               (if capture-output?
-                                 #(binding [test/report report]
-                                    (capture/with-test-buffer
-                                      (test/test-var v)))
-                                 #(binding [test/report report]
-                                    (test/test-var v))))))
+                              (binding [report/*testing-path* [ns ::test/each-fixtures]]
+                                (try
+                                  (each-fixtures
+                                    (if capture-output?
+                                      #(binding [test/report report
+                                                 report/*testing-path* [ns v]]
+                                          (capture/with-test-buffer
+                                            (test/test-var v)))
+                                      #(binding [test/report report
+                                                 report/*testing-path* [ns v]]
+                                          (test/test-var v))))
+                                  (catch Throwable t
+                                    (test/do-report {:type :error
```

Comment:
We can probably factor out the exception into a function since we're using it twice:

```clojure
(defn- uncaught-exception [throwable]
  {:type    :error
   :message "Uncaught exception during fixture initialization."
   :actual  throwable})
```

That'll prevent the line from exceeding 80 characters as well.